README.md

```diff
@@ -1,38 +1,43 @@
-# Cuvis C# examples
+![image](https://camo.githubusercontent.com/9fc396a08b84779ea0f78a4085e96bee6035fca702cd382f38cb661fa1ff1d0c/68747470733a2f2f7777772e7370656374726f6578706f2e636f6d2f77702d636f6e74656e742f75706c6f6164732f323031382f30372f637562657274323031382e706e67)
 
-## Building
-To the the exampels running, first clone this git repository and initialize it's submodules recursively
+
+# cuvis.csharp.examples
+
+## Building the examples
+To get the exampels running, first set up a project directory, clone this git repository and initialize its submodule(s).
 
 ```
-git submodule update --init --recursive
```

Comment:
I am pretty sure it needs to be recursive

Reply:
was a copy paste.